projects/schematics/src/migrations/4_0/removed-public-api-deprecations/removed-public-api-deprecation.ts

```diff
@@ -34,6 +35,19 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
   //   importPath: SPARTACUS_STOREFRONTLIB,
   //   comment: `'${VARIANT_STYLE_SELECTOR_COMPONENT}' was moved to @spartacus/product/variants.`,
   // },
+
+  // projects/core/src/smart-edit/smart-edit.module.ts
+  {
+    node: 'SmartEditModule',
```

Comment:
make these into constants as done above in other files `projects/schematics/src/shared/constants.ts`